src/csharp/Microsoft.Spark/Sql/Types/SimpleTypes.cs

```diff
@@ -69,21 +69,52 @@ public sealed class BooleanType : AtomicType
     /// </summary>
     public sealed class DateType : AtomicType
     {
-        private static readonly DateTime s_unixTimeEpoch = new DateTime(1970, 1, 1);
+        internal static readonly DateTime s_unixTimeEpoch = new DateTime(1970, 1, 1);
```

Comment:
I think we can just set this to `new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);` and use from `TimeStamp` as well.

Reply:
Sure, I have changed all to use `DateType.s_unixTimeEpoch`.